sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -32,6 +32,12 @@ class JoinSuite extends QueryTest with SharedSQLContext {
 
   setupTestData()
 
+  override def afterEach(): Unit = {
```

Comment:
yes, this is a default behavior. 
But it can explain the work to clear the cache.
```
 // Clear the cache table for test cases
 // by super.afterEach()
```